packages/graphql/src/schema/make-augmented-schema.ts

```diff
@@ -88,7 +88,13 @@ function makeAugmentedSchema(
         enableRegex,
         skipValidateTypeDefs,
         generateSubscriptions,
-    }: { enableRegex?: boolean; skipValidateTypeDefs?: boolean; generateSubscriptions?: boolean } = {}
+        callbacks,
+    }: {
+        enableRegex?: boolean;
+        skipValidateTypeDefs?: boolean;
+        generateSubscriptions?: boolean;
+        callbacks?: Neo4jGraphQLCallbacks;
```

Comment:
Yeah, this being littered everywhere is definitely bugging me. The fact that it's both here and also in the context shows that we have some serious duplicity here. Perhaps we should have a discussion about this before merging?